PPAK_CVS/Sources/Scenes/Product/ProductVC.swift

```diff
@@ -51,28 +71,47 @@ final class ProductViewController: BaseViewController, Viewable {
 
   override func setupLayouts() {
     super.setupLayouts()
+    [navigationHeaderBarView, collectionView].forEach {
+      view.addSubview($0)
+    }
 
-    // navigation bar
-    navigationController?.navigationBar.topItem?.rightBarButtonItem = UIBarButtonItem(
-      image: UIImage(systemName: "heart"),
-      style: .plain,
-      target: self,
-      action: nil
-    )
-
-    navigationItem.titleView = UIImageView(frame: CGRect(x: 0, y: 0, width: 100, height: 50)).then {
-      $0.backgroundColor = .systemGray
+    [backButton, featureStackView].forEach {
+      navigationHeaderBarView.addSubview($0)
     }
 
-    // add views
-    view.addSubview(collectionView)
+    [bookmarkButton, shareButton].forEach {
+      featureStackView.addArrangedSubview($0)
+    }
   }
 
   override func setupConstraints() {
     super.setupConstraints()
 
+    navigationHeaderBarView.snp.makeConstraints { make in
+      make.top.horizontalEdges.equalTo(view.safeAreaLayoutGuide)
+      make.height.equalTo(60)
+    }
+
+    backButton.snp.makeConstraints { make in
+      make.centerY.equalToSuperview()
+      make.leading.equalToSuperview().inset(16)
+      make.size.equalTo(44)
+    }
+
+    featureStackView.snp.makeConstraints { make in
+      make.centerY.equalToSuperview()
+      make.trailing.equalToSuperview().inset(16)
+    }
+
+    [bookmarkButton, shareButton].forEach {
+      $0.snp.makeConstraints { make in
+        make.size.equalTo(44)
+      }
+    }
+
     collectionView.snp.makeConstraints { make in
-      make.edges.equalTo(view.safeAreaLayoutGuide)
+      make.horizontalEdges.bottom.equalTo(view.safeAreaLayoutGuide)
+      make.top.equalTo(navigationHeaderBarView.snp.bottom)
```

Comment:
collectionview bottom 수정 부탁드려요~
```suggestion
      make.top.equalTo(navigationHeaderBarView.snp.bottom)
      make.horizontalEdges.bottom.equalToSuperview()
```